### PR TITLE
[MM-61459] Fix errcheck issues in server/channels/app/plugin_signatur…

### PR DESCRIPTION
**Pull request by molejnik88**
…e.go

#### Ticket Link

Fixes https://github.com/mattermost/mattermost/issues/29074
Jira https://mattermost.atlassian.net/browse/MM-61459

#### Release Note

```release-note

```

### CODE REVIEW

File: server/channels/app/plugin_signature.go [hunk 1]
```diff
@@ -90,8 +90,10 @@ func (ch *Channels) verifyPlugin(plugin, signature io.ReadSeeker) *model.AppErro
 			continue
 		}
 		publicKey := bytes.NewReader(pkBytes)
-		plugin.Seek(0, 0)
-		signature.Seek(0, 0)
+		//nolint:errcheck // safe operation: setting offset to the start of the reader
+		plugin.Seek(0, io.SeekStart)
+		//nolint:errcheck // safe operation: setting offset to the start of the reader
+		signature.Seek(0, io.SeekStart)
```
Comment (hanzei) [inline server/channels/app/plugin_signature.go:96]: Why are you not checking the error that `Seek` returns?

Comment (molejnik88) [reply server/channels/app/plugin_signature.go]: It seemed to be a safe operation. I assumed that the "0" offset does not exceed the seize of the underlying object.
But I decided to log errors and continue the for-loop.

**Review by hanzei — APPROVED** [review]
LGTM :+1:

**Review by gabrieljackson — APPROVED** [review]

**Comment by hanzei** [comment]
Thanks @molejnik88 :tada:

**Pull request merged**